tests/lwip/tcp.c

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ * @}
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+
+#include "common.h"
+#include "od.h"
+#include "net/af.h"
+#include "net/sock/tcp.h"
+#include "net/ipv6.h"
+#include "thread.h"
+#include "xtimer.h"
+
+#ifdef MODULE_SOCK_TCP
+static char sock_inbuf[SOCK_INBUF_SIZE];
+static bool server_running, client_running = false;
```

Comment:
only client running will be set to false, which is probably not what you intended. Anyhow, if declared like this, the default is zero (false) anyways.